packages/shared/src/hooks/useAdImpressions.ts

```diff
@@ -1,7 +1,6 @@
 import { isSameDay } from 'date-fns';
 import usePersistentState from './usePersistentState';
 import { Ad } from '../graphql/posts';
-import { logRevenue, gaTrackEvent } from '../lib/analytics';
```

Comment:
Please check if we need this file at all. I'm not sure we need this hook anymore

Reply:
Ah yes looks like we don't need this at all as it only kept the count, but we don't do anything with that now.